daemon/info.go

```diff
@@ -174,7 +175,13 @@ func (daemon *Daemon) showPluginsInfo() types.PluginsInfo {
 
 	pluginsInfo.Volume = volumedrivers.GetDriverList()
 	pluginsInfo.Network = daemon.GetNetworkDriverList()
-	pluginsInfo.Authorization = daemon.configStore.AuthorizationPlugins
+
+	authPlugins := []string{}
```

Comment:
slice needs to be a continuous memory address, then if the capacity is not enough, runtime needs to re-allocate the memory and move the original slice data to new address. As a result, it may lead to some kind of performance issue. Is this one of the reasons?